### PR TITLE
Fixed issues in mobile devices and gadget maximization

### PR DESCRIPTION
**Pull request by lasanthaS**

### CODE REVIEW

File: apps/portal/extensions/components/gadget/index.js [hunk 1]
```diff
@@ -93,11 +93,11 @@
                     params[name] = option.value;
                 }
             }
-
+
```
Comment (TanyaM) [inline apps/portal/extensions/components/gadget/index.js]: Remove new line

File: apps/portal/js/ues-dashboards.js [hunk 2]
```diff
@@ -176,6 +176,19 @@
         $('.ues-component-box').each(function (i, container) {
             container = $(container);
             
+            // Calculate the data-height field which is required to render the gadget
+            if (isDesigner) {
+
```
Comment (TanyaM) [inline apps/portal/js/ues-dashboards.js]: remove new line

File: apps/portal/extensions/components/gadget/index.js (hunk 1 above)
Comment (lasanthaS) [inline apps/portal/extensions/components/gadget/index.js]: This is fixed

File: apps/portal/js/ues-dashboards.js (hunk 2 above)
Comment (lasanthaS) [inline apps/portal/js/ues-dashboards.js]: Fixed in the PR https://github.com/wso2/carbon-dashboards/pull/174

**Pull request merged**